src/bundle/Resources/config/dashboard.yml

```diff
@@ -0,0 +1,87 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    EzSystems\HybridPlatformUi\Dashboard\:
+        resource: '../../../lib/Dashboard/*'
+
+    EzSystems\HybridPlatformUi\Dashboard\Dashboard:
+
+    EzSystems\HybridPlatformUi\Dashboard\Section\MyContent:
+        calls:
+            - [setTemplate, ['EzSystemsHybridPlatformUiBundle:dashboard:my_content_section.html.twig']]
+            - [setParameter, ['identifier', 'my_content']]
+        tags:
+            - name: ezplatform.ui.dashboard.section
+              identifier: my_content
+              position: 1
```

Comment:
`position` name is something to discuss (maybe we should name it `priority` here?). It works like `priority` used in Symfony. Lower `position` will appear first, those with the same position will appear in an order of definition, higher obviously later.

Here, every element has it own position unique to his location:
- `1, 10, 20` for tabs in `my_content` section,
- `1, 10` for tabs in `all_content` section
- and `1, 10` to sections `my_content` and `all_content` (respectively).